src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -15,8 +15,9 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
   const createExecutiveEducationFailureMessage = (failureCode) => {
     const failureCodeMessages = {
       no_offer_available: 'No offer is available to cover this course.',
-      no_offer_with_enough_balance: 'Your organization doesn’t have sufficient balance to cover this course.',
-      no_offer_with_enough_user_balance: 'You don’t have sufficient balance to cover this course.',
+      no_offer_with_enough_balance: 'Your enrollment was not completed! Your organization does not have remaining credit. Please contact your administrator in regards to this limit.',
```

Comment:
The "Please contact your administrator..." messaging is now duplicative of the following text below:

```jsx
<p className="mb-6">
  Please contact your edX administrator to resolve the error and gain access to this content.
</p>
```

That said, both of these messages are soon changing in Markhors' redesign of these error screens ([here](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/705/files#diff-6ae4418eeb69028fea14051ec26a812584a3743d36d3a003c300741004854189R23) and [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/705/files#diff-6ae4418eeb69028fea14051ec26a812584a3743d36d3a003c300741004854189L33-L34)).